app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -1,19 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::Inventory::Persister::AutomationManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::AutomationManager
+
+  has_inventory automation_manager.credentials
```

Comment:
also not sure about ICs that point to other ICs

https://github.com/Ladas/manageiq-providers-amazon/blob/80c25bac4a54198d45ce04a52e2bf2349a41c7ef/app/models/manageiq/providers/amazon/inventory/persister/cloud_manager.rb#L12-L12